test/integration/editor/index.spec.ts

```diff
@@ -192,3 +192,18 @@ test('code editor auto-complete', async ({ page }) => {
   await page.keyboard.type('textF');
   await expect(page.getByRole('option', { name: 'textField' })).toBeVisible();
 });
+
+test('must deselect selected element when clicking outside of it', async ({ page }) => {
+  const editorModel = new ToolpadEditor(page);
+
+  await editorModel.goToPageById('K7SkzhT');
+
+  await editorModel.waitForOverlay();
+  const nodeHudTag = editorModel.appCanvas.getByTestId('node-hud-tag');
+  const input = editorModel.appCanvas.locator('input:nth-child(1)');
```

Comment:
The reason I said we can use a text field was to make it full-width, so that the selection and the element have the same size.
Can you make the `fullWidth` prop be `true` for this text field?